extension/chrome/elements/compose-modules/formatters/general-mail-formatter.ts

```diff
@@ -80,4 +89,19 @@ export class GeneralMailFormatter {
     // throws ComposerResetBtnTrigger when user closes pass phrase dialog without entering
     return await view.storageModule.decryptSenderKey(parsedSenderPrv);
   };
+
+  private static isSenderInKeyUserIds = async (view: ComposeView, signingKey?: ParsedKeyInfo | undefined): Promise<boolean> => {
+    const senderEmail = view.senderModule.getSender();
+    const storage = await AcctStore.get(view.acctEmail, ['sendAs']);
+    const emailAliases = Object.keys(storage.sendAs || {});
```

Comment:
these 2 lines can be moved inside `if` statement, as we don't need to fetch aliases when `if` check returns `false`

Reply:
Right, thanks!